fe/fe-core/src/test/java/org/apache/doris/cloud/cache/CacheHotspotManagerTest.java

```diff
@@ -23,11 +23,11 @@
 import org.apache.doris.catalog.Tablet;
 import org.apache.doris.cloud.CacheHotspotManager;
 import org.apache.doris.cloud.system.CloudSystemInfoService;
+import org.apache.doris.common.Triple;
 import org.apache.doris.system.Backend;
 
 import mockit.Mock;
 import mockit.MockUp;
-import org.apache.commons.lang3.tuple.Triple;
```

Comment:
just ask: what's wrong with the original package? @kaijchen

Reply:
for gson support